tests/scripts/setup-pytest-env.sh

```diff
@@ -74,17 +75,14 @@ function run_pytest() {
 
     suite_name="${test_suite_name}-${current_shard}-${ffi_type}"
 
-    # Some test environments don't play well with parallelism
-    DEFAULT_PARALLELISM=2
-    if [[ "${TEST_STEP_NAME:-default}" == "frontend: GPU"* ]] || [[ "${TEST_STEP_NAME:-default}" == "test: Hexagon"* ]]; then
-        DEFAULT_PARALLELISM=1
-    fi
+    DEFAULT_PARALLELISM=1
```

Comment:
The n=2 wasn't actually being used (the `if` below was always failing) and caused some CI issues that should be resolved in a follow up